tests/integ/sagemaker/jumpstart/script_mode_class/test_transfer_learning.py

```diff
@@ -96,13 +103,19 @@ def test_jumpstart_transfer_learning_estimator_class(setup):
         instance_count=instance_count,
         instance_type=training_instance_type,
         metric_definitions=default_metric_definitions,
+        **estimator_kwargs,
```

Comment:
question: should we remove from these `kwargs` all the arguments defined above? Otherwise wouldn't we risk overriding customer's inputs?

Reply:
generally speaking, yes. but for this integration test, since we know there are no conflicts, we should be good. In the implementation of `JumpStartModel` and `JumpStartPredictor` we will check for conflicts so that customer inputs are respected